script/optimization_concepts/conic_programming.jl

```diff
@@ -220,10 +220,10 @@ model = Model(optimizer_with_attributes(ECOS.Optimizer, "printlevel" => 0))
 @variable(model, t[1:n])
 @variable(model, x[1:n])
 @objective(model, Max, sum(t))
-@constraint(model, sum(x) == 1)
+@constraint(model, sum(x) == 1.0)
 @constraint(model, A * x .<= b )
 # Cannot use the exponential cone directly in JuMP, hence we use MOI to specify the set.
-@constraint(model, con[i = 1:n], [1, x[i], t[i]] in MOI.ExponentialCone())
+@constraint(model, con[i = 1:n], [t[i], x[i], 1.0] in MOI.ExponentialCone())
 
 optimize!(model);
 

```

Comment:
Maybe we should add a test on the objective value to verify the optimality of the solution found

Reply:
That makes sense.

Reply:
I add new test only for the different conic formulations. But that would be great to refact the tests overall in my opinion.

Reply:
you mean in the file or in the tutorials overall?

Reply:
in the tutorial overall

Reply:
Just catch a typo in the tests for `SecondOrderCone` formulation (my bad). I will fix that ASAP.